x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/components/rules_table/add_prebuilt_rules_table/add_prebuilt_rules_table_context.tsx

```diff
@@ -232,10 +233,21 @@ export const AddPrebuiltRulesTableContextProvider = ({
         {previewedRule && (
           <RuleDetailsFlyout
             rule={previewedRule}
-            actionButtonLabel={i18n.INSTALL_BUTTON_LABEL}
-            isActionButtonDisabled={canPreviewedRuleBeInstalled}
-            onActionButtonClick={installOneRule}
+            dataTestSubj="installPrebuiltRulePreview"
             closeFlyout={closeRulePreview}
+            ruleActions={
+              <EuiButton
+                disabled={canPreviewedRuleBeInstalled}
+                onClick={() => {
+                  installOneRule(previewedRule.rule_id ?? '');
+                  closeRulePreview();
+                }}
+                fill
+                data-test-subj="installPrebuiltRuleButton"
```

Comment:
To differentiate this from the button to install a single rule without previewing, what do you think about replacing this like so:
```
data-test-subj={`installPrebuiltRuleFromFlyoutButton-${ruleId}`}
```
in the same way you can see in `x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/components/rules_table/add_prebuilt_rules_table/use_add_prebuilt_rules_table_columns.tsx`

(Not sure if it makes sense to pass in the `ruleId` in this case, since there should always be one these elements in the DOM at any time, right?)

Reply:
Makes sense, will change the `data-test-subj`. I think it should be fine without the `id` since there's only one install / upgrade button on the screen at a time.